ibm/service/vpc/resource_ibm_is_vpc_routing_table_test.go

```diff
@@ -206,3 +224,50 @@ resource "ibm_is_vpc_routing_table" "test_ibm_is_vpc_routing_table" {
 	accept_routes_from_resource_type=["%s"]
 }`, name, rtName, acceptRoutesFromVPNServer)
 }
+
+func testAccCheckIBMISVPCRouteTableAdvertiseRoutesToConfig(rtName, name, acceptRoutesFromVPNServer, advertiseRoutesTo1, advertiseRoutesTo2 string) string {
+	return fmt.Sprintf(`
+resource "ibm_is_vpc" "testacc_vpc" {
+	name = "%s"
+}
+resource "ibm_is_vpc_routing_table" "test_ibm_is_vpc_routing_table" {
+	depends_on = [ibm_is_vpc.testacc_vpc]
+	route_direct_link_ingress = true
+	route_transit_gateway_ingress = true
+	vpc = ibm_is_vpc.testacc_vpc.id
+	name = "%s"
+	accept_routes_from_resource_type=["%s"]
+	advertise_routes_to=["%s","%s"]
+}`, name, rtName, acceptRoutesFromVPNServer, advertiseRoutesTo1, advertiseRoutesTo2)
+}
+
+func testAccCheckIBMISVPCRouteTableAdvertiseRoutesToDLConfig(rtName, name, acceptRoutesFromVPNServer, advertiseRoutesTo1 string) string {
+	return fmt.Sprintf(`
+resource "ibm_is_vpc" "testacc_vpc" {
+	name = "%s"
+}
+resource "ibm_is_vpc_routing_table" "test_ibm_is_vpc_routing_table" {
+	depends_on = [ibm_is_vpc.testacc_vpc]
+	route_direct_link_ingress = true
+	route_transit_gateway_ingress = true
+	vpc = ibm_is_vpc.testacc_vpc.id
+	name = "%s"
+	accept_routes_from_resource_type=["%s"]
+	advertise_routes_to=["%s"]
+}`, name, rtName, acceptRoutesFromVPNServer, advertiseRoutesTo1)
+}
+
+func testAccCheckIBMISVPCRouteTableAdvertiseRoutesToRemovalConfig(rtName, name string) string {
+	return fmt.Sprintf(`
+resource "ibm_is_vpc" "testacc_vpc" {
+	name = "%s"
+}
+resource "ibm_is_vpc_routing_table" "test_ibm_is_vpc_routing_table" {
+	depends_on = [ibm_is_vpc.testacc_vpc]
+	route_direct_link_ingress = true
+	vpc = ibm_is_vpc.testacc_vpc.id
+	name = "%s"
+	accept_routes_from_resource_type=[]
+	advertise_routes_to=[]
```

Comment:
We need to set explictly to empty array ? what happens if user remove the argument itself from tf file advertise_routes_to

Reply:
It is a optional and computed field. so if user removes the argument itself, then there won't be any change shown. 
user has to explicitly set empty array.